ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.html

```diff
@@ -1,31 +1,24 @@
 <main class="main">
     <section class="section">
-        <div class="content">
-            <h1>First, enter your invitation code and zip code</h1>
-            <p>If you received a paper or email invitation from your doctor,
-                it includes a 12-digit invitation code at the bottom.</p>
+        <div class="content registration-block">
             <form class="invitation-form" [formGroup]="formGroup" (ngSubmit)="onSubmit()">
-                <mat-form-field class="invitation-form__input" floatLabel="always">
-                    <mat-label>Enter invitation code</mat-label>
-                    <input matInput
-                           invitationcode
-                           formControlName="invitationId"
-                           pattern="[a-zA-Z0-9]{12}"
-                           maxLength="18"
-                           placeholder="Your invitation code">
+                <h1 class="invitation-form__title semi-bold">First, confirm your eligibility</h1>
+                <div class="invitation-form__container">
+                    <p class="invitation-form__question bold">Enter invitation code</p>
+                    <img class="invitation-form__info" src="assets/images/info.png" alt="Info icon"
+                        matTooltip="If you received a paper or email invitation from your doctor, it includes a 12-digit invitation code at the top."
+                        matTooltipPosition="right">
+                </div>
+                <mat-form-field class="invitation-form__input">
+                    <input matInput invitationcode formControlName="invitationId" pattern="[a-zA-Z0-9]{12}"
+                        maxLength="18" placeholder="Enter invitation code">
                 </mat-form-field>
-                <mat-form-field class="invitation-form__input" floatLabel="always">
-                    <mat-label>What is your current mailing zip code?</mat-label>
-                    <input matInput
-                           formControlName="zip"
-                           minlength="5"
-                           maxlength="5"
-                           placeholder="5 digit zip"
-                           pattern="[0-9]{5}">
+                <p class="invitation-form__question bold">What is your current mailing zip code?</p>
+                <mat-form-field class="invitation-form__input" Label="always">
+                    <input matInput formControlName="zip" minlength="5" maxlength="5"
+                        placeholder="Enter mailing zip code" pattern="[0-9]{5}">
```

Comment:
very minor thing. I have a preference to laying out the attributes in elements like this vertically. I find it a lot easier to read and notice what is going on. Also easier to do merges.

Reply:
Autoformatting code betrayed me :)